pyproject.toml

```diff
@@ -24,11 +24,14 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.8 || ^3.9 || ^3.10 || ^3.11 || ^3.12"
```

Comment:
Don't be lazy

Reply:
![image](https://github.com/user-attachments/assets/acfd0125-7c57-4057-b606-a50ed469376c)